src/functions.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+$booksDb = connectToDb('booksDb');
+
+$books = getAllBooks($booksDb);
+
+function displayBooks($books) {
+    $htmlOutput = ''; 
+    foreach ($books as $book) {
+        if (substr($book['authorSecondName'], -1) == "s") {
+            $possessiveMarker = "'";
+        } else {
+            $possessiveMarker = "'s";
+        }    
+        $link = $book['link'];
+        $htmlOutput .= "<section tabindex='0'>";
+        $htmlOutput .= "<h2>" . $book['bookTitle'] . "</h2>";
+        $htmlOutput .= "<p tabindex='0' class='quote'>&#8220" . $book['keyQuote'] . "&#8221</p>";
+        $htmlOutput .= "<p tabindex='0'><span class='bookTitle'>" . $book['bookTitle'] 
+            . "</span> is a " 
```

Comment:
indentation here is out and for all below

Reply:
I've changed the indent but not sure if correct